tofu/README.md

```diff
@@ -170,7 +170,7 @@ The application backend is deployed as an ECS service via terragrunt
 
 #### Frontend
 
-The application frontend is deployed to an S3 bucket fronted by Cloudfront.  As such it can be deployed by deploying files to S3 and creating a Cloudfront invalidation.
+The application frontend is deployed to an S3 bucket fronted by Cloudfront.  As such it can be deployed by uploadingx files to S3 and creating a Cloudfront invalidation.
```

Comment:
```suggestion
The application frontend is deployed to an S3 bucket fronted by Cloudfront.  As such it can be deployed by uploading files to S3 and creating a Cloudfront invalidation.
```
Typo